app/assets/javascripts/start-modules.js

```diff
@@ -1,9 +1,9 @@
-//= require govuk/modules
-//= require modules/global-bar
-//= require modules/sticky-element-container
-//= require modules/toggle
-//= require modules/track-click
+// = require govuk/modules
```

Comment:
I don't know about all of them, but toggle definitely still works in the front end.